wordpress-mobile/WordPress-iOS
# Issues/atomic plugin fix

This PR hides the autoupdate toggle for all plugins installed on an atomic site.  This is to match changes introduced in calypso. For more background see p9o2xV-87-p2

To test:
On an atomic site, install the AMP plugin.  Confirm you see the autoupdate toggle in the app.
Switch to this branch. 
Check the plugin again and confirm that the autoupdate toggle is hidden.

@jklausa may I trouble you for a review?
